.github/workflows/test.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: test
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22'
+      - name: yarn install
+        run: yarn install
+      - name: Run tests
+        run: yarn test
```

Comment:
Can you add large_test  as well

Reply:
And can you check if you can update the node addon api and node gyp latest to the same version as fcntl if it works

Reply:
done, the update on the node addon and node gyp worked as well.